catalog-info.yaml

```diff
@@ -212,3 +212,42 @@ spec:
 # ***********************************
 # SECTION END: DRA pipelines
 # ***********************************
+
+# ********************************************
+# Declare supported plugin tests pipeline
+# ********************************************
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: 'Logstash supported plugins test pipeline'
```

Comment:
I am very sorry to have misled you here, but after a deeper look it appears that the metadata name for a backstage resource can't include spaces. [Here](https://backstage.io/docs/features/software-catalog/descriptor-format/#name-required) is the spec which mentions:

- Strings of length at least 1, and at most 63
- Must consist of sequences of `[a-z0-9A-Z]` possibly separated by one of `[-_.]`

Reply:
Fixed with 867246f4bc9ad5a4b1fbbebff0abd0b727aed25d